### PR TITLE
ansible: account_disable_post_pw_expiration

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

**Comment by shawndwells** [comment]
@openscap-ci test this please

File: shared/templates/static/ansible/account_disable_post_pw_expiration.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: Set Account Expiration Following Inactivity
+  lineinfile:
+    create: yes
+    dest: /etc/default/useradd
+    regexp: ^INACTIVE
+    line: INACTIVE=(ansible-populate var_disable_post_pw_expiration)
```
Comment (yuumasato) [inline shared/templates/static/ansible/account_disable_post_pw_expiration.yml]: variable should be:  `var_account_disable_post_pw_expiration`

**Comment by shawndwells** [comment]
@yuumasato yikes - thanks! updated.

**Comment by yuumasato** [comment]
Jenkins has given green light: https://jenkins.open-scap.org/view/SCAP%20Security%20Guide/job/scap-security-guide-pull-requests/217/.
Ack.

**Pull request merged**